frontend/src/metabase/admin/settings/auth/components/AuthCard/AuthCard.unit.spec.tsx

```diff
@@ -79,11 +79,10 @@ describe("AuthCard", () => {
   });
 });
 
-const getSetting = (opts?: Partial<AuthSetting>): AuthSetting =>
-  createMockSettingDefinition({
-    value: false,
-    ...opts,
-  }) as AuthSetting;
+const getSetting = (opts?: Partial<AuthSetting>): AuthSetting => ({
+  value: false,
```

Comment:
I'd add some real authentication setting key here, so we don't need to modify `frontend/src/metabase/admin/settings/auth/components/AuthCard/AuthCard.tsx`

```suggestion
  key: "google-auth-enabled",
  value: false,
```

Reply:
@WiNloSt I'm still getting type errors - 
![image](https://github.com/user-attachments/assets/4fd7130e-cfde-4daf-92e1-922f512ac07a)

```Type 'SettingDefinition<"admin-email" | "email-smtp-host" | "email-smtp-port" | "email-smtp-security" | "email-smtp-username" | "email-smtp-password" | "enable-embedding" | "enable-embedding-static" | ... 109 more ... | "slack-token-valid?">' is not assignable to type 'AuthSetting'.
  Type 'SettingDefinition<"admin-email" | "email-smtp-host" | "email-smtp-port" | "email-smtp-security" | "email-smtp-username" | "email-smtp-password" | "enable-embedding" | "enable-embedding-static" | ... 109 more ... | "slack-token-valid?">' is not assignable to type '{ value: boolean | null; }'.
    Types of property 'value' are incompatible.
      Type 'SettingValue<"admin-email" | "email-smtp-host" | "email-smtp-port" | "email-smtp-security" | "email-smtp-username" | "email-smtp-password" | "enable-embedding" | "enable-embedding-static" | ... 109 more ... | "slack-token-valid?">' is not assignable to type 'boolean | null'.
        Type 'undefined' is not assignable to type 'boolean | null'.ts(2322)
```

this is why I made these changes in the first place - I don't know if we can actually do this without extra pain that I don't think is warranted for one test

Reply:
Well, I couldn't figure out a cleaner way for this either. Let's do what you think is best in this case then.